src/util.rs

```diff
@@ -300,22 +318,39 @@ pub fn print_blocks(
     Ok(())
 }
 
-pub fn color_from_rgba(
+pub fn color_from_rgb(
```

Comment:
Removed alpha since it wasn't being used?

Reply:
Yes i always was confused about it because the `i3bar` protocol doesn't even support alpha. It was never used.

Reply:
I think `i3-gaps` (https://github.com/Airblader/i3/blob/22059fc6f580462a81be0fbbc28821344362ce88/i3bar/src/main.c#L63) and `sway` both support the alpha channel though.

Reply:
Looks like vanilla i3 also has that option: https://github.com/i3/i3/blob/eb83b37936d5356291f49fdc6c1d23ac02fd1c5a/i3bar/src/main.c#L63

Reply:
https://classicforum.manjaro.org/index.php?topic=27138.0

Reply:
Huh I didn't know that, will revert later

Reply:
Reverted